cypress/support/fragments/inventory/holdingsRecordView.js

```diff
@@ -0,0 +1,34 @@
+import inventoryInstance from './inventoryInstance';
+import { Button, Layer } from '../../../../interactors';
+import QuickMarcEditor from '../quickMarcEditor';
+import InventoryViewSource from './inventoryViewSource';
+
+const root = Layer('View holdings record');
+const actionsButton = root.find(Button('Actions'));
+const closeButton = root.find(Button({ icon: 'times' }));
+const editInQuickMarcButton = Button({ id: 'clickable-edit-marc-holdings' });
+const viewSourceButton = Button({ id: 'clickable-view-source' });
+
+export default {
+  newHolding : {
+    rowsCountInQuickMarcEditor : 6
```

Comment:
Why it hardcoded? Is it possible to put this value from test? or is it just a constant?

Reply:
Because I don't have option to read actual rows count into quickmarc editor. I set related TODO into code and will back to this in the future